core/input/input.h

```diff
@@ -85,6 +85,8 @@ class Input : public Object {
 
 		virtual bool has_joy_light() const { return false; }
 		virtual bool set_joy_light(const Color &p_color) { return false; }
+		virtual bool has_joy_adaptive_triggers() const { return false; }
+		virtual bool send_joy_packet(const void *p_data, int p_size) { return false; }
```

Comment:
```suggestion
		virtual bool send_joy_packet(const void *p_data, size_t p_size) { return false; }
```
That would be better, I think